cmd/load-smoke-data/load-smoke-data.go

```diff
@@ -212,20 +212,6 @@ func main() {
 	const dbURI = "mongodb://localhost:27017"
```

Comment:
(random comment placement to allow for threading)

I'm assuming that part of the work for this ticket will be to remove the static username/password stored on the host? Can we please document how to connect to the hosts locally after this change in our internal docs?

Reply:
ah yes, my bad, all good there